unified-runtime/source/adapters/level_zero/image_common.cpp

```diff
@@ -274,7 +274,15 @@ ur_result_t createUrImgFromZeImage(ze_context_handle_t hContext,
                                    ur_exp_image_mem_native_handle_t *pImg) {
   ze_image_handle_t ZeImage;
   ZE2UR_CALL(zeImageCreate, (hContext, hDevice, &ZeImageDesc, &ZeImage));
-  ZE2UR_CALL(zeContextMakeImageResident, (hContext, hDevice, ZeImage));
+  if (ZeImage == nullptr) {
```

Comment:
Is it actually possible for ZeImage to be nullptr if we got SUCCESS from zeImageCreate?